Formula/dash.rb

```diff
@@ -4,8 +4,9 @@ class Dash < Formula
   license "BSD-3-Clause"
 
   stable do
-    url "http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.11.1.tar.gz"
-    sha256 "73c881f146e329ac54962766760fd62cb8bdff376cd6c2f5772eecc1570e1611"
+    url "http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.11.2.tar.gz"
+    mirror "https://dl.bintray.com/homebrew/mirror/dash-0.5.11.2.tar.gz"
+    sha256 "00fb7d68b7599cc41ab151051c06c01e9500540183d8aa72116cb9c742bd6d5f"
 
     # Fix compilation on MacOS
```

Comment:
Is this patch still needed?

Reply:
yeah, not needed, just did the local check.